gst/datarepo/gstdatareposink.c

```diff
@@ -39,7 +39,7 @@
 /**
  * @brief Tensors caps
  */
-#define TENSOR_CAPS GST_TENSORS_CAP_MAKE ("{ static, flexible }")
+#define TENSOR_CAPS GST_TENSORS_CAP_MAKE ("{ static, flexible, sparse }")
```

Comment:
Is there any reason to use MACRO?

Reply:
Thank you for your comment!
When creating the element's pad template, the MACRO already existed and I just used it.
MACRO has format and framerate, It also seems to reduce redundant code.